docs/source/docker/advanced/datasource_metadata.rst

```diff
@@ -0,0 +1,177 @@
+.. _ref-docker-datasource-metadata:
+
+Add Metadata to a Datasource
+===============================
+
+Lightly can make use of metadata collected alongside your images or videos. Provided
+metadata can be used to steer the selection process and to analyze the selected subset
+in the web-app.
+
+
+Metadata Folder Structure
+----------------------------
+
+In the following, we will outline the format in which metadata can be added to a
```

Comment:
in the following, we will outline the format -> Following, we outline the structure and format